### ISSUE TITLE
FUJITSU Split Klimageräte

### ISSUE THREAD

**Issue by Tagiwara** [ioBroker/AdapterRequests#763]
**What kind of device or service you would like to see an adapter for?**
FUJITSU Split Klimageräte

**Is the device connected to the internet or only in the local network available?**
Wifi Modul ist vorhanden

**Is an official App or Website available?**
Die App heißt "FGLAir"

**Is an official API including documentation available?**
nicht bekannt

**Are other libraries for an integration available?**
nicht bekannt

**Is this device already integrated in other Smart Home systems?**
nicht bekannt

**Is this device already integrated in homebridge? Might the ham adapter together with the homebridge plugin be sufficient?**
nicht bekannt

**Additional context**
Add any other context or screenshots about the feature request here. If the topic was discussed in ioBroker forum please include the link too.

After you created the issue vote for yourself in the first post of the issue using the "+1"/"Thumbs up" button

**Comment by michaw192** [comment]
Besteht die möglichkeit einen adapter zu bauen damit ich die app airstage aich von iobroker bedienen kann.

**Comment by michaw192** [comment]
Welche Informationen fehlen ?

**Comment by mcm1957** [comment]
Ausser der Ingo dass eine App existiert konnte von den Fragen im ersten Beitrag nichts beantwortet werden, insbesondere nichtvob ein Api existiert.

Generell fehlt es wahrscheinlich aber eher an einrm Entwickler.

**Comment by michaw192** [comment]
> Ausser der Ingo dass eine App existiert konnte von den Fragen im ersten Beitrag nichts beantwortet werden, insbesondere nichtvob ein Api existiert.
> 
> Generell fehlt es wahrscheinlich aber eher an einrm Entwickler.

Diese Frage kann ich leider auch nicht genau beantworten allerdings lässt aufgrund der Tatsache dass es eine Alexa Skill gibt darauf schließen dass es eine gibt.

**Comment by Apollon77** [comment]
> lässt aufgrund der Tatsache dass es eine Alexa Skill gibt darauf schließen dass es eine gibt.

Nope, leider nicht. Eher die Tatsache das es eine APp gibt lässt darauf schliessen das die auch eine API hat :-)

**Comment by jtolksdo** [comment]
Es gibt einen connector für homeassistant hier:
https://github.com/danielkaldheim/ha_airstage

**Comment by mcm1957** [comment]
Repo: https://github.com/stefan5232/ioBroker.fujitsu-airstage
Latest PR https://github.com/ioBroker/ioBroker.repositories/pull/6070
